lib/sum_of_multiples.rb

```diff
@@ -0,0 +1,22 @@
+class SumOfMultiples
+  @number
+  @another_number
+  def initialize(a, b)
+    @number = a
+    @another_number = b
+  end
+  def calculate_sum(limit)
+    i = 1
+    sum=0
+    loop do
+      if i >= limit
```

Comment:
Use if-else only if you think it is not possible to implement solution without it. https://williamdurand.fr/2013/06/03/object-calisthenics